java-util/src/main/java/io/druid/java/util/common/CompressionUtils.java

```diff
@@ -313,7 +319,7 @@ public InputStream openStream() throws IOException
    *
    * @return A GZIPInputStream that can handle concatenated gzip streams in the input
    */
-  public static GZIPInputStream gzipInputStream(final InputStream in) throws IOException
+  private static GZIPInputStream gzipInputStream(final InputStream in) throws IOException
```

Comment:
This can break extensions that depend on it. Can it be deprecated for a version or two instead?